README.md

```diff
@@ -69,3 +69,45 @@ app.post('/contacts/:contactid', function (req, res) {
 	});
 });
 ```
+
+### Proposed item function reference
+Via the use of optional parameters and dedicated functions it is possible to retrieve anything from a field on a single record to all fields on all records. Selection criteria and response timeouts may also be specified.
+
+* Return all records of a type that match an optional criteria
+  + jsonArray = cmdb.getAllItems(locals, type, criteria = None, timeout = 6000)
```

Comment:
I assume by jsonArray/jsonObject you mean native javascript arrays and objects?  Not a json string which needs decoding...